Makefile

```diff
@@ -21,10 +24,15 @@ package:
 	python setup.py bdist
 
 format:
-	black -t py36 -l 100 gpflow tests doc setup.py
+	black $(BLACK_CONFIG) $(BLACK_TARGETS)
+
+format-check:
+	black --check $(BLACK_CONFIG) $(BLACK_TARGETS)
 
 type-check:
-    mypy .
+	mypy .
 
-test:
+pytest:
```

Comment:
i think `test` is better. it expresses what it does, whilst not commit to a particular test runner

Reply:
oh, I just saw you've redefined `test` ... one sec

Reply:
ok. my opinion, which I hold lightly, is that type checking isn't part of testing. I also don't think formatting is part of testing

Reply:
e.g. `check-all`